tap_gorgias/client.py

```diff
@@ -1,12 +1,14 @@
 import json
 import requests
 import requests.exceptions
+import os
 import singer
 import time
 import urllib
 
 LOGGER = singer.get_logger()
 
+DEFAULT_TIMEOUT = os.getenv('DEFAULT_HTTP_TIMEOUT')
```

Comment:
Do we also want to set a default here in case the env variable gets removed/doesn't get it for whatever reason?

Reply:
Yea, by default it will always just to default to None (which is the default input for the timeout anyway). Do you think it's better then to auto-default to 10 instead?

`DEFAULT_TIMEOUT = os.getenv('DEFAULT_HTTP_TIMEOUT')`

Reply:
I guess the main argument against adding this default within this file is that its easier to rollback if we don't include it here. Like if adding a timeout was causing the tap to fail instead of updating the logic here, updating the tag and pushing the changes, we would only have to modify the variable in the Dockerfile for lloyd and merge that?

Reply:
Yes that's a valid argument for easier rollbacks. I think this is fine for now then